### ISSUE TITLE
Query : How to draw square shaped markers in line graph.

### ISSUE THREAD

**Issue by anishparajuli555** [danielgindi/Charts#1405]
I am trying to change the rounded markers to square shape as shown in the image below..But didnot help..how can i acheive this effect?
<img width="726" alt="screen shot 2016-09-06 at 9 33 14 pm" src="https://cloud.githubusercontent.com/assets/7752578/18280722/c9f3cdf8-7479-11e6-8785-270db56377bc.png">

**Comment by liuxuan30** [comment]
it's `private func drawCircles(context context: CGContext)`
However seems time to support more shapes for line chart. Scatter chart does support more shapes

**Comment by danielgindi** [comment]
@PhilJay I'm thinking about reusing the `ShapeRenderer`s for LineChart

**Comment by PhilJay** [comment]
Yep, that's not such a bad idea. I have not thought through all differences yet, but maybe we can remove the `ScatterChart` altogether.

If `LineChart` can handle shapes as well, we might just need to add an option to toggle the lines.

But not totally sure if it's a good idea :-)

**Comment by dvertovsek** [comment]
Any news on this? Any tips how to do it by myself?

I guess, I would have to play around with the `private func drawCircles(context context: CGContext)`
which is `line 598` inside the `LineChartRenderer.swift` ?
